src/Pages/Login/DetailPage/DetailPage.js

```diff
@@ -23,13 +26,33 @@ class DetailPage extends Component {
       })
       .then(res => {
         this.setState({
-          detail: res.job_detail
+          detail: res.job_detail,
+          response_rate: res.job_detail.response_rate
         });
+        if (this.state.response_rate > 50) {
+          this.setState({ button: true });
+        }
       });
   }
+  handleChangeColor = () => {
+    if (this.state.color === "#fe415c") {
+      this.setState({ color: "#dbdbdb", likes_count: 0 });
+    } else {
+      this.setState({
+        color: "#fe415c",
+        likes_count: 1
+      });
+    }
```

Comment:
삼항연산자로 표현하면 더 가독성에 좋을 것 같네요! 
```suggestion
    this.state.color === "#fe415c" ? 
      this.setState({ color: "#dbdbdb", likes_count: 0 }) : this.setState({ color: "#fe415c", likes_count: 1 });
    
```